python/ray/_private/usage/usage_lib.py

```diff
@@ -356,18 +360,23 @@ def _generate_cluster_metadata():
     return metadata
 
 
-def show_usage_stats_prompt() -> None:
+def show_usage_stats_prompt(cli: bool) -> None:
     if not usage_stats_prompt_enabled():
         return
 
     from ray.autoscaler._private.cli_logger import cli_logger
 
+    prompt_print = cli_logger.print if cli else print
+
     usage_stats_enabledness = _usage_stats_enabledness()
     if usage_stats_enabledness is UsageStatsEnabledness.DISABLED_EXPLICITLY:
-        cli_logger.print(usage_constant.USAGE_STATS_DISABLED_MESSAGE)
+        prompt_print(usage_constant.USAGE_STATS_DISABLED_MESSAGE)
     elif usage_stats_enabledness is UsageStatsEnabledness.ENABLED_BY_DEFAULT:
-
-        if cli_logger.interactive:
+        if not cli:
```

Comment:
It seems this path is only hit for nightly wheels. Consider renaming the cli flag and also the message to `is_nightly` / `...ENABLED_BY_DEFAULT_FOR_NIGHTLY_MESSAGE` for clarity.

Reply:
It's more of cli vs ray.init rather than nightly vs non-nightly. Nightly cli and non-nightly cli are the same.